ndarbas14/iveskAutomobili.php

```diff
@@ -0,0 +1,128 @@
+<?php
+session_start();
+require_once 'Radar.php';
+require_once 'Filtras.php';
+require_once 'startRadarSes.php';
+
+if (!isset($_SESSION['automobiliai'])) {
+    $_SESSION['automobiliai'] = [];
+
+}
+
+if (!isset($filtr)) {
+    if (isset($_POST['filtras'])) {
+        $filtr = new Filtras($_POST['filtras']);
+    }
+    else {
+        $filtr = new Filtras('');
+    }
+}
+
+if ($_SERVER['REQUEST_METHOD'] =='POST') {
+    if (isset($_POST['data'], $_POST['numeris'], $_POST['atstumas'], $_POST['laikas']) && strlen($_POST['data']) > 0) {
+        $_SESSION['automobiliai'][] = [
+            $_POST['data'],
+            $_POST['numeris'],
+            $_POST['atstumas'],
+            $_POST['laikas']
+        ];
+    }
+
+    if (isset($_POST['filtras']))  {
+        $filtr->setFiltroReiksme($_POST['filtras']);
+    }
+    else {
+        $filtr->setFiltroReiksme('');
+    }
+
+}
+
+if (!isset($duomenys)) {
```

Comment:
Nereikalingas tikrinimas.

Reply:
Išmetus tikrinimą if (!isset($duomenys)), naujai užkraunant puslapį metamas pranešimas:
Warning: usort() expects parameter 1 to be array, null given in C:\wamp64\www\03_paskaita_20180212\iveskAutomobili.php on line 48

Reply:
`$duomenys = []` palikit, ismeskit tik visa tikrinima.